bower.json

```diff
@@ -17,6 +17,10 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-prelude": "^3.0.0"
+    "purescript-prelude": "^3.0.0",
+    "purescript-foldable-traversable": "^3.3.0"
```

Comment:
Yes, I think that's the simplest approach. It's pretty imperative, but then so is `Eff` generally. Something specific to `Eff` like

```purescript
class Functor f <= Iterable f where
  forEach_ :: forall eff a b. (a -> Eff eff b) -> f a -> Eff eff Unit
  forEach :: forall eff a b. (a -> Eff eff b) -> f a -> Eff eff (f b)
```

should provide a nice sweet spot between "too general, instances can't be made efficient" and "too specific, not enough instances".